server/bin/gold/test-18.txt

```diff
@@ -0,0 +1,223 @@
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/BACKED-UP/tarball_config-02_1970.01.01T00.00.41.tar.xz
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/TO-RE-UNPACK/tarball_config-02_1970.01.01T00.00.41.tar.xz
+
+Restored 2 (0 existing) of (3): /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz
+
+
+cp -a /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz.md5 /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/
+md5sum --check /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz.md5
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/BACKED-UP/tarball_config-00_1970.01.01T00.00.40.tar.xz
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/TO-RE-UNPACK/tarball_config-00_1970.01.01T00.00.40.tar.xz
+
+Restored 3 (0 existing) of (3): /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz
+
+Restored 3 tar balls from backup volume, /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup (ended at: 42.0, elapsed (0.00)
+--- Finished pbench-restore-and-unpack-tarballs (status=0)
++++ Running pbench-unpack-tarballs none re-unpack
+Template:  pbench-unittests.v3.server-reports
+Index:  pbench-unittests.v3.server-reports.1970-01 1
+len(actions) = 1
+[
+    {
+        "_id": "5ca1ab1e70015f100dedfab1ed0ff1ce",
+        "_index": "pbench-unittests.v3.server-reports.1970-01",
+        "_op_type": "create",
+        "_source": {
+            "@generated-by": {
+                "commit_id": "unit-test",
+                "group_id": 43,
+                "hostname": "example.com",
+                "pid": 42,
+                "user_id": 44,
+                "version": ""
+            },
+            "@timestamp": "1970-01-01T00:00:42",
+            "chunk_id": 1,
+            "doctype": "status",
+            "name": "pbench-re-unpack-tarballs",
+            "text": "pbench-re-unpack-tarballs.run-1970-01-01T00:00:42-UTC(unit-test) - w/ 0 errors\nProcessed 0 result tar balls, 0 successfully, 0 warnings, 0 errors, and 0 duplicates\n\n",
```

Comment:
Didn't it process three tar balls? NM, it was a dry run... assuming IUC.

Reply:
Yes, I believe that '18 is a dry run...which puts another facet on an issue that I pointed out earlier:  the dry run should have indicated that it _would have_ processed three (or whatever) tarballs...but didn't.  And, the fact that it _didn't_ suggests that it shouldn't be making a report about it.

Reply:
This is expected.  In the case of a dry-run, `pbench-restore-and-unpack-tarballs` does not take any action, so the cron job, `pbench-re-unpack-tarballs` does not process any thing.